hybrid/hybrid_simulation.py

```diff
@@ -414,6 +414,15 @@ def simulate(self,
 
         logger.info(f"Hybrid Simulation complete. NPVs are {self.net_present_values}. AEPs are {self.annual_energies}.")
 
+    @property
+    def system_capacity_kw(self):
+        cap = self.outputs_factory.create()
+        for v in ("pv", "wind", "battery"):
```

Comment:
Can we make this for loop based on power_sources so it will include future technologies?

Reply:
Good point! adding now